src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,116 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int stSize = 10;
+    private int realSize = 0;
+    private T[] items;
+
+    public ArrayList() {
+        items = (T[]) new Object[stSize];
+    }
+
+    private void resize() {
+        if (realSize >= stSize) {
+            stSize *= 1.5;
```

Comment:
[All magic numbers should be constant](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/array-list.html#any-magic-numbers-should-be-constants)